src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do List!</title>
+</head>
+<body>
+  <h1>To Do List!!</h1>
+  <!-- img alt="" src=<%=require("./img/Rectangle55.png")%> / -->
```

Comment:
Please avoid having unnecessary lines in your codebase, for example, comment lines like this one.

Reply:
😜I'll remove it, it's my codebase for adding an image in HTML with Webpack😉